Jenkinsfile.integration

```diff
@@ -24,6 +24,8 @@ pipeline {
         /* first value in the list is the default */
         choice(choices: ['airship', 'osh'], description: 'Which deployment mechanism?', name: 'deployment')
         choice(choices: ['caasp3', 'caasp4'], description: 'Which caasp version to use?', name: 'caasp_version')
+        booleanParam(name: 'run_tempest', defaultValue: true, description: 'Run tempest tests?')
+        choice(choices: ['smoke', 'all'], description: 'Run smoke or all tempest tests?', name: 'tempest_test_type')
```

Comment:
Would you want a default value for the smoke vs all choices?

Reply:
on `choices` first option is the default. Not very intuitive...

Reply:
Yes, I agree.

I will change default for run_tempest to false as @Itxaka mentioned, we want to avoid running tempest for PR change, even waiting for related automation changes to merge